src/main/webapp/definitions/alfresco-model.yaml

```diff
@@ -114,27 +114,43 @@ paths:
 
             The following where clause will only return aspects from the `namespace1:model` and `namespace2:model`.
               ```
-              where=(modelIds in ('namespace1:model','namespace2:model'))
+              where=(modelId in ('namespace1:model','namespace2:model'))
+              where=(modelId in ('namespace1:model INCLUDESUBASPECTS','namespace2:model'))
               ```
 
             The following where clause will only return sub aspects for the given parents.
               ```
-              where=(parentIds in ('namespace1:parent','namespace2:parent'))
+              where=(parentId in ('namespace1:parent','namespace2:parent'))
               ```
 
             The following where clause will only return aspects that match the pattern.
               ```
-              where=(namespaceUri matches('http://www.alfresco.org/model.*'))
+              where=(namespaceUri matches('http://www.alfresco.*'))
               ```
 
             The following where clause will only return aspects that don't match the pattern.
               ```
-              where=(not namespaceUri matches('http://www.alfresco.org/model.*'))
+              where=(not namespaceUri matches('http://www.alfresco.*'))
               ```
           required: false
           type: string
         - $ref: '#/parameters/skipCountParam'
         - $ref: '#/parameters/maxItemsParam'
+        - name: include
+          in: query
+          description: |
+            Returns additional information about the aspect. The following optional fields can be requested:
+            * properties
+            * container
```

Comment:
We typically use the include mechanism for data that could be expensive to retrieve or make the response very large, properties, associations and mandatoryAspects are good examples of this. 

Personally I would just have boolean properties in the response by default.